constellations/orion/src/domains/family.go

```diff
@@ -0,0 +1,33 @@
+package domains
+
+import (
+	"database/sql"
+	"errors"
+	"regexp"
+	"time"
+)
+
+var TABLE_FAMILY = "family"
```

Comment:
can we call the table `families`?

Also where are the migration files? the up.sql and the down.sql files?